external/linux/include/imconfig.h

```diff
@@ -0,0 +1,123 @@
+//-----------------------------------------------------------------------------
```

Comment:
Why have these ImGui files been duplicated into the `external/linux/` folder?

Reply:
See https://github.com/cg-tuwien/Gears-Vk/pull/109#commitcomment-54032014 and https://github.com/cg-tuwien/Gears-Vk/pull/109#commitcomment-54032270